invenio_pages/assets/semantic-ui/js/invenio_pages/search/SearchResultsContainer.js

```diff
@@ -0,0 +1,67 @@
+/*
+ * This file is part of Invenio.
+ * Copyright (C) 2023 CERN.
+ *
+ * Invenio is free software; you can redistribute it and/or modify it
+ * under the terms of the MIT License; see LICENSE file for more details.
+ */
+
+import PropTypes from "prop-types";
+import React from "react";
+import { Table } from "semantic-ui-react";
+import isEmpty from "lodash/isEmpty";
+import { i18next } from "@translations/invenio_administration/i18next";
+
+export const SearchResultsContainer = ({
+  results,
+  columns,
+  displayEdit,
+  displayDelete,
+  actions,
+}) => {
+  const resourceHasActions = displayEdit || displayDelete || !isEmpty(actions);
+
+  const columnWidth = {
```

Comment:
I am not sure about this approach, it's i) hardcoding field names and ii) ties the front-end rendering to the current state of the backend. E.g. in the future the representation of a page might change and this code won't work anymore. 

However I don't have any suggestion, if anyone else wants to suggest feel free

Reply:
Unless we adjust the width of each column according to the size of the content of all the cells, I don't think there is a better way to define the size of the columns. I also don't think is that problematic to hardcode the field names, as this is not a generic module but an override. Finally, even do it's true that we are tying the front-end rendering to the current-state of the backend, without doing it an going for generic widths like in the base invenio-administration, the column distribution looks wrong.

Reply:
I just saw an alternative in anika's PR so I will go with it to avoid defining here the columnWidths.